driver/src/test/java/org/neo4j/driver/integration/RoutingDriverBoltKitIT.java

```diff
@@ -18,108 +18,35 @@
  */
 package org.neo4j.driver.integration;
 
-import org.junit.jupiter.api.AfterEach;
-import org.junit.jupiter.api.BeforeAll;
 import org.junit.jupiter.api.Test;
-import reactor.core.publisher.Flux;
-import reactor.core.publisher.Mono;
-import reactor.test.StepVerifier;
-
-import java.io.IOException;
-import java.net.URI;
-import java.util.concurrent.TimeUnit;
 
 import org.neo4j.driver.Config;
 import org.neo4j.driver.Driver;
 import org.neo4j.driver.GraphDatabase;
-import org.neo4j.driver.Record;
 import org.neo4j.driver.net.ServerAddress;
 import org.neo4j.driver.net.ServerAddressResolver;
-import org.neo4j.driver.reactive.RxResult;
-import org.neo4j.driver.reactive.RxSession;
-import org.neo4j.driver.util.StubServer;
-import org.neo4j.driver.util.StubServerController;
 
-import static org.hamcrest.core.IsEqual.equalTo;
-import static org.hamcrest.junit.MatcherAssert.assertThat;
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertThrows;
 import static org.mockito.ArgumentMatchers.any;
 import static org.mockito.Mockito.mock;
 import static org.mockito.Mockito.verify;
 import static org.mockito.Mockito.when;
-import static org.neo4j.driver.SessionConfig.builder;
-import static org.neo4j.driver.util.StubServer.insecureBuilder;
+import static org.neo4j.driver.Logging.none;
 
-/**
- * New tests should be added to testkit (https://github.com/neo4j-drivers/testkit).
- *
- * This class exists only for the following:
- * - to keep the remaining tests that are due to be migrated
- * - to keep the tests that are currently not portable
- */
-@Deprecated
 class RoutingDriverBoltKitIT
```

Comment:
Maybe we should rename this one to something like `ResolverFailureIT` or perhaps theres a suitable class to shift it to